tasks/main.yml

```
@@ -241,7 +241,7 @@
 
     - name: Ensure Sonar is started.
       wait_for:
-        path: /usr/local/sonar/logs/sonar.log
+        path: "/usr/local/sonar/logs/sonar.{{ ansible_facts.ansible_date_time.year}}{{ ansible_facts.ansible_date_time.month}}{{ ansible_facts.ansible_date_time.day}}.log"
```

Why don't you access `ansible_date_time` directly?

```suggestion
        path: "/usr/local/sonar/logs/sonar.{{ ansible_date_time.year}}{{ ansible_date_time.month}}{{ ansible_date_time.day}}.log"
```